packages/shared/src/modules/categories/libs/enums/categories-validation-message.enum.ts

```diff
@@ -0,0 +1,5 @@
+const categoriesValidationMessage = {
+	NOT_INTEGER_ARRAY: "Query's value should be a stringified array of integers",
```

Comment:
```suggestion
	NOT_INTEGER_ARRAY: "Query's value should be a stringified array of integers.",
```
We have dots in all other `...-validation-message.enum.ts`'s.
Also it is not a user-friendly message. Non-dev related person won't understand it.
Take a look at this comment:
https://github.com/BinaryStudioAcademy/bsa-2024-bebalance/pull/208#discussion_r1739616976

Reply:
There are no dots for user validation messages. I will add the dot to this one.